rclcpp/include/rclcpp/callback_group.hpp

```diff
@@ -55,19 +56,19 @@ class CallbackGroup
 
   RCLCPP_PUBLIC
   const std::vector<rclcpp::subscription::SubscriptionBase::WeakPtr> &
-  get_subscription_ptrs() const;
+  get_subscription_ptrs();
```

Comment:
I don't think that that's possible when the functions are locking a mutex that is a member of the class.